foundry.toml

```diff
@@ -8,7 +8,11 @@ ffi = true
 fs_permissions = [{ access = "read-write", path = ".forge-snapshots/"}, { access = "read", path = "./foundry-out"}]
 evm_version = 'paris'
 
-[profile.ci]
-fuzz_runs = 100000
+[profile.default.fuzz]
+runs = 100
```

Comment:
why are the runs different in default vs ci?

Reply:
So it runs faster locally! Lots of runs can make it quite slow esp with lot of fuzz tests

Reply:
So it runs faster locally! Lots of runs can make it quite slow esp with lot of fuzz tests